go.mod

```diff
@@ -62,6 +61,7 @@ require (
 	github.com/dlclark/regexp2 v1.9.0 // indirect
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
 	github.com/go-redis/redis/v8 v8.11.5 // indirect
+	github.com/golang/snappy v0.0.4 // indirect
```

Comment:
Now, we should do the same with the Prometheus remote write so we can drop it.

Reply:
Just make certain to test it as it might turn out there are incompatibilities ... although I hope not :crossed_fingers:

Reply:
Added a issue there